.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,22 +1,26 @@
 <!-- Thanks for contributing to _fastlane_! Before you submit your pull request, please make sure to check the following boxes by putting an x in the [ ] (don't: [x ], [ x], do: [x]) -->
 
 ### Checklist
+
```

Comment:
I added these line breaks as it's a markdown good practice to surround headers with line breaks (before and after), as some markdown interpreters don't parse them correctly when there are no line breaks after headers